### ISSUE TITLE
Images aren't shown correctly in inverted rendering mode

### ISSUE THREAD

**Issue by FrankEular** [anthonycr/Lightning-Browser#673]
Hello,

Thanks for the development and the maintenance. I found that in the inverted rendering mode, the images aren't shown properly. Take https://en.wikipedia.org/wiki/Hypercovering as an example: these mathematical formulae are rendered invisible.

Possible solution: add a rendering mode, inverting not only texts but also colors in images.

**Comment by jacko9000** [comment]
Perhaps the solution could be implemented through stylesheet processing, in order to produce a black background and white text, with images and videos being left untouched.

**Comment by FrankEular** [comment]
Personally I think that it might be better to furnish choices between different rendering modes, like https://addons.mozilla.org/en-US/firefox/addon/dark-background-light-text/

**Comment by jacko9000** [comment]
I agree, although that add-on does in fact use stylesheet processing in its default mode, as it is both the fastest and most effective mode.

**Comment by pdimri** [comment]
Any fix for the video issue by dev?
